_posts/2025-01-09-struct-decode-intro.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: "Structured Decoding in vLLM: a gentle introduction"
+author: "Guest Post by BentoML Team"
```

Comment:
This should be co-authored by BentoML and Red Hat

Reply:
Should I /s/Neural Magic/Red Hat?

Reply:
Just Red Hat! thanks!

The blog looks great